# Prevent accidental check-in of network throttling (no-browser-throttle)

## Rule Details

Examples of **incorrect** code for this rule:

```js
await browser.throttleOn();
await browser.throttleOff();
```
